### PR TITLE
Fixed Pull Request

### PR DESCRIPTION
**Pull request by Zakhrov**
Hi, I had commited some credentials by mistake in my earlier fork.
This one is clean and able to merge

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-ruby '2.5.0'
+ruby '2.6.3'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '6.0.0'
 # Use sqlite3 as the database for Active Record
 # gem 'sqlite3'
-# Use Puma as the app server
-gem 'puma', '~> 3.11'
+# Use Passenger as the app server
```
Comment (michelson) [inline Gemfile]: revert comment on Passenger

File: Gemfile.lock [hunk 2]
```diff
@@ -363,7 +364,7 @@ GEM
       nokogiri (~> 1.6)
       rubyzip (~> 1.0)
       selenium-webdriver (>= 3.0, < 4.0)
-    webpacker (4.0.7)
+    webpacker (3.5.0)
```
Comment (michelson) [inline Gemfile.lock]: webpacker downgrade ?

**Comment by michelson** [comment]
please check travis CI , it fails on ruby version. can't merge this. I would suggest multiple PR along all the changes you made. ruby version, react transition group, ... in order to review & merge fast

best

File: Gemfile.lock (hunk 2 above)
Comment (Zakhrov) [reply Gemfile.lock]: Yes. Webpacker 4.0 was giving me asset compilation errors

**Comment by Zakhrov** [comment]
I have sent a new PR with an updated travis.yml and a more clear changelog

**Pull request closed without merging**